README.md

```diff
@@ -28,7 +28,7 @@ Create a new WebSocket connection using the following.
 - Password is optional.  
 
 ```js
-const obs = new OBSWebSocket('address', 'password');
+const obs = new OBSWebSocket({ address: 'localhost:4444', password: '$up3rSecretP@ssw0rd' });
```

Comment:
Gotcha, that sucks that it'll have to wait a couple months.

Total aside, but couldn't shake the thought today that you and Support Class could try to reach out to aid the official Magic the Gathering stream. I know a guy on their production crew that regularly handles 1000+ person events and last I knew from him they were doing their overlays and graphics with Powerpoint and a chroma key in the latter half of 2016. Something to look into perhaps if you guys need a target.